### PR TITLE
Clarify taxation and dividends

### PR DESCRIPTION
**Pull request by someknave**
There are some interactions between taxation from module 3 and dividends in module 9, rules for these interactions were clarified in [this bgg thread](https://boardgamegeek.com/thread/1471947/dividends-vs-taxation-93x-and-39x).
Extra line added to dividends when playing with 3.I/II to make players record each company's income, so that taxation can be calculated the next round.
Extra line added to taxation when playing with 9.I to clarify the case where the company doesn't have enough assets to pay taxes. (shouldn't be an issue for 9.II)
Some other clarifications to taxes with 9.I/II were added.

Latest commit was fixing the example for taxes in 73x and 93x so that an extra cell is created for the example in those cases and only in those cases. I welcome someone else clarifying this further or changing the formatting as they feel appropriate.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -606,9 +608,9 @@
 
 	{{#capture}}
 		{{#3}}{{^3.III}}{{^9.I}}{{^7.I}}
-			<p>At game start, all players make a secret bid with their starting money ($0 is a valid bid). Starting with the highest bidder, each player chooses a position in the player order and his capital on the map. In case of a tie, randomly draw lots for the order of choice (the losers get $2 for each lost draw).</p>
+			<p>At game start, all players make a secret bid with their starting money ($0 is a valid bid). {{^9.II}}Starting with the highest bidder, each player chooses a position in the player order and his capital on the map.{{/9.II}}{{#9.II}}{{/9.II}} In case of a tie, randomly draw lots for the order of choice (the losers get $2 for each lost draw).</p>
```
Comment (Arachnid) [inline index.html]: Why is there an empty #9.II section here?

File: index.html [hunk 2]
```diff
@@ -637,7 +639,10 @@
 			<div class="col-md-2 col-xs-2"><h1>Phase 1: Company Order</h1></div>
 			<div class="col-md-10 col-xs-10">
 				<p>The order of the stock companies matches the actual share values from high to low. In case of a tie the order of the concerned companies matches the stack of share value markers from top to bottom.</p>
-				{{#9.I}}{{#3.II}}<p>Starting with the 2nd round the income is taxed according to the companies with most to least income: The company with the highest income from the previous round pays 20% tax, the company with the with the lowest pays 0%, and all other players pay 10% of their last income, see above for <b>Calculation of taxes.</b> In the case of a tie, the company earlier in the turn order pays the higher tax rate. The company turn order continues to be determined according to the actual share value.</p>{{/3.II}}{{/9.I}}
+				{{#3.II}}
```
Comment (Arachnid) [inline index.html:642]: This will now show on all games with module 3.II, instead of just those with 9.I as well. Is that intentional?

Comment (someknave) [inline index.html:642]: The # 9.I was redundant this is already within #9 and  ^9.III.

Comment (Arachnid) [inline index.html:642]: Good point, thanks.

File: index.html (hunk 1 above)
Comment (someknave) [inline index.html]: Good catch, I was struggling with the rules trying to work out how this combination worked and must have left this for later. I'll fix this now.

**Comment by Arachnid** [comment]
Thanks again!

**Pull request merged**